### PR TITLE
Add named collection grant description

### PR DESCRIPTION
**Pull request by jsc0218**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit cd2a418afd088563819ed723d7a1ae91110cb895 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60893/cd2a418afd088563819ed723d7a1ae91110cb895/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/sql-reference/statements/grant.md [hunk 1]
```diff
@@ -498,6 +505,23 @@ and
 [`format_display_secrets_in_show_and_select` format setting](../../operations/settings/formats#format_display_secrets_in_show_and_select)
 are turned on.
 
+### NAMED COLLECTION ADMIN
```
Comment (kssenii) [inline docs/en/sql-reference/statements/grant.md:508]: Worth to add a note tha before 23.7 it was called NAMED_COLLECTION_CONTROL, and after 23.7 it is preserved as an alias.

File: docs/en/sql-reference/statements/grant.md [hunk 2]
```diff
@@ -498,6 +505,23 @@ and
...
+
+Allows a certain operation on a specified named colleciton.
+
+- `NAMED COLLECTION ADMIN`. Level: `NAMED_COLLECTION`. Aliases: `NAMED COLLECTION CONTROL`
+    - `CREATE NAMED COLLECTION`. Level: `NAMED_COLLECTION`
+    - `DROP NAMED COLLECTION`. Level: `NAMED_COLLECTION`
+    - `ALTER NAMED COLLECTION`. Level: `NAMED_COLLECTION`
+    - `SHOW NAMED COLLECTIONS`. Level: `NAMED_COLLECTION`. Aliases: `SHOW NAMED COLLECTIONS`
+    - `SHOW NAMED COLLECTIONS SECRETS`. Level: `NAMED_COLLECTION`. Aliases: `SHOW NAMED COLLECTIONS SECRETS`
+    - `NAMED COLLECTION`. Level: `NAMED_COLLECTION`. Aliases: `NAMED COLLECTION USAGE, USE NAMED COLLECTION`
```
Comment (kssenii) [inline docs/en/sql-reference/statements/grant.md:518]: Worth to mention that unlike all other grants (CREATE, DROP, ALTER, SHOW) grant NAMED_COLLECTION was added only in 23.7, while all other were added earlier - in 22.12.

Comment (jsc0218) [reply docs/en/sql-reference/statements/grant.md:518]: both done

File: docs/en/sql-reference/statements/grant.md [hunk 3]
```diff
@@ -498,6 +505,25 @@ and
 [`format_display_secrets_in_show_and_select` format setting](../../operations/settings/formats#format_display_secrets_in_show_and_select)
 are turned on.
 
+### NAMED COLLECTION ADMIN
+
+Allows a certain operation on a specified named collection. Before version 23.7 it was called NAMED COLLECTION CONTROL, and after 23.7 NAMED COLLECTION ADMIN as added and NAMED COLLECTION CONTROL is preserved as an alias.
```
Comment (kssenii) [inline docs/en/sql-reference/statements/grant.md:510]:
```suggestion
Allows a certain operation on a specified named collection. Before version 23.7 it was called NAMED COLLECTION CONTROL, and after 23.7 NAMED COLLECTION ADMIN was added and NAMED COLLECTION CONTROL is preserved as an alias.
```

**Review by kssenii — APPROVED** [review]

**Pull request merged**